packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -244,6 +245,13 @@ const baseDesignTokens = {
     lg: 700,
     xl: 800,
   },
+  iconSize: {
+    xs: '0.75rem',
+    sm: '1rem',
+    md: '1.25rem',
+    lg: '1.5rem',
+    xl: '1.75rem',
+  },
```

Comment:
@hbjORbj Can you share what are you planing to use this in Avatar component?

Reply:
For `size` prop of `Avatar`. I will make it return only 3 sizes from sm to lg.